public/app.d.ts

```diff
@@ -2641,6 +2641,97 @@ declare namespace JMap {
          */
         function remove(listenerId: string): void
       }
+
+      /**
+       * ***JMap.Application.Event.Extension***
+       * 
+       * Here you can manage all JMap NG App extension event listeners.
+       * 
+       * Click to see all events available: ***[[JMap.Application.Event.Extension.on]]***. 
+       */
+      namespace Extension {
+
+        /**
+         * ***JMap.Application.Event.Extension.on***
+         * 
+         * Here you have all JMap NG App available extension events on which you can attach a listener.
+         */
+         namespace on {
+
+          /**
+           * ***JMap.Application.Event.Extension.on.registration***
+           * 
+           * This event is triggered after a user registers his own JMap NG App extension.
+           * 
+           * The extension id is received as a param property of the event.
```

Comment:
J'ai retiré ", you can then take any action that would be executed on double click for this layer."